spec/services/reports/children_served_report_service_spec.rb

```diff
@@ -65,8 +65,8 @@
       expect(report).to eq({
         name: 'Children Served',
         entries: {
-          'Average children served monthly' => "10.0",
```

Comment:
I don't know if I can just changes these values to make the test pass. It appears to me that that the logic is sound and the test was written without the default distribution_quantity in mind. Thoughts?